.github/workflows/ci.yml

```diff
@@ -428,25 +428,3 @@ jobs:
         run: |
           git status --porcelain
           test -z "$(git status --porcelain)"
-
-  build-java:
```

Comment:
does this mean we don't build or test the Java integration in CI?

Reply:
we do, the java workflow should trigger on pr (see the checks on this PR).

the only step that doesn't run is the publish